sdk/python/feast/online_response.py

```diff
@@ -77,3 +77,46 @@ def to_df(self, include_event_timestamps: bool = False) -> pd.DataFrame:
         """
 
         return pd.DataFrame(self.to_dict(include_event_timestamps))
+
+
+class OnlineResponseRow(OnlineResponse):
```

Comment:
Is this used anywhere? I couldn't find any usage.

Reply:
Oh nice catch, I accidentally copied this over. Will delete.